astropy/utils/diff.py

```diff
@@ -43,7 +43,7 @@ def diff_values(a, b, rtol=0.0, atol=0.0):
         return a != b
 
 
-def report_diff_values(a, b, fileobj=sys.stdout, indent_width=0):
+def report_diff_values(a, b, fileobj=sys.stdout, indent_width=0, rtol=0.0, atol=0.0):
```

Comment:
Is this the current software default for fitsdiff? Seems unnecessarily strict.

Reply:
I guess so:
https://github.com/astropy/astropy/blob/d3b64b9b85edd8f11732c86851e075c08dd8f636/astropy/io/fits/diff.py#L199-L202